pkg/sdk/functions_validations_gen.go

```diff
@@ -101,6 +133,12 @@ func (opts *AlterFunctionOptions) validate() error {
 	if !ValidObjectIdentifier(opts.name) {
 		errs = append(errs, ErrInvalidObjectIdentifier)
 	}
+	if opts.RenameTo != nil && !ValidObjectIdentifier(opts.RenameTo) {
+		errs = append(errs, ErrInvalidObjectIdentifier)
```

Comment:
in cases other than name validation I would recommend using errInvalidIdentifier function, because right now by error message we wouldn't know if the invalid identifier is opts.name or opts.RenameTo

Reply:
fixed